ChegeBryan/politico
# #1639333451 Separate test dummy data

#### What does this PR do?

Separates office and party test data into separate modules

#### Description of Task to be completed?

- [x] Move office test data into separate files
- [x] Move party test data into separate files

#### How should this be manually tested?
After cloning the repository to your local machine - check [README.md](README.md) for instructions. Move to the repository directory and check out  `ch-separate-dummy-data-1639333451`. The test folder will have the test data dummy in separate modules.

#### Any background context you want to provide?
Makes the code more readable.

#### What are the relevant pivotal tracker stories?
[#1639333451](https://www.pivotaltracker.com/story/show/1639333451)
